README.md

```diff
@@ -23,6 +23,26 @@ rails db:setup
 # This runs db:create, db:schema:load, and db:seed
```

Comment:
what version of postgresql? (let's make sure this version that is listed on readme is the same that is used for production on heroku)

Reply:
👍

Reply:
heh i think there might be an added decimal

Reply:
cool good catch.